README.md

```diff
@@ -1 +1,28 @@
-# actions
\ No newline at end of file
+## Fastlane GitHub Actions
+
+The Fastlane GitHub Actions provide a set of GitHub Actions to make maintaining projects easier.
+
+## Actions
+
+- 🌏 [@github-actions/fastlane-env-reminder](fastlane-env-reminder)
+
+   Adds a comment to include _fastlane_ environment information in an issue description if it is missing. Read more [here](fastlane-env-reminder).
+   
+- ✔️ [@github-actions/communicate-on-pull-request-merged](communicate-on-pull-request-merged)
```

Comment:
This link does not work yet.

Reply:
That's correct. Do you think we should remove it from the docs then? 🤔 The PR is ready and I hope it will be merged pretty soon, so I'd suggest to leave it 🚀

Reply:
Nah, then this is perfectly fine.